src/hooks/useNewsRoomPaging.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+
+import { GetMoreNews } from "../services/ApiService";
+
+const useNewsRoomPaging = props => {
```

Comment:
you can probably get rid of this file. Since this is almost a duplicate of getnews, we can just pass some parameters to `useNewsRoom({ page, perPage});` which will happen in a later pr.

Reply:
which in return will also require us to update GetNews in the ApiServices file

Reply:
Will make these changes once we tackle the load more task